swap/swap_test.go

```diff
@@ -736,6 +738,9 @@ func TestDebtCheques(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	testBackend.cashDone = make(chan struct{})
```

Comment:
please initialize this channel inside in `newTestBackend` when you return the struct:
```go
return &swapTestBackend{
		TestBackend:    backend,
		factoryAddress: factoryAddress,
		tokenAddress:   tokenAddress,
                cashDone:         make(chan struct{}),
}
```

Reply:
done